### DISCUSSION TITLE
小爱音箱mini没有MediaPlayer实体和对话记录实体

### DISCUSSION THREAD

**Discussion by smile-wingbow** [XiaoMi/ha_xiaomi_home#444] [Ideas]
能否像Xiaomi Miot Auto一样加上MediaPlayer和对话记录这两个实体？

**Comment by SusanPhevos** [comment]
v0.3.1 之后可生成 Media Player 实体。
xiaomi_home 不会支持查询音箱对话历史。https://github.com/XiaoMi/ha_xiaomi_home/wiki/Features-that-will-not-be-implemented#wifi-speaker-conversation-history
